src/site/content/en/lighthouse-accessibility/aria-allowed-attr/index.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: Ensure ARIA attributes are allowed for an element's role
+description: |
+  Learn about aria-allowed-attr audit.
+author: megginkearney
+web_lighthouse:
+  - aria-allowed-attr
+---
+
+[ARIA](https://www.w3.org/TR/wai-aria-1.1/#role_definitions)
+roles and attributes help screen readers
+provide missing information about an element.
+For these roles and attributles to make sense,
+each ARIA `role` supports a specific subset of `aria-*` attributes
+(see [ARIA roles definitions](https://www.w3.org/TR/wai-aria-1.1/#role_definitions)).
+Lighthouse reports any mismatches between roles and `aria-*` attributes:
+
+<figure class="w-figure">
+  <img class="w-screenshot w-screenshot--filled" src="aria-allowed-attr.png" alt="Lighthouse: ARIA attributes do not match their roles">
+  <figcaption class="w-figcaption">
+    Fig. 1 — ARIA attributes do not match their roles
+  </figcaption>
+</figure>
+
+## About ARIA
```

Comment:
Duplicating the ARIA sections is interesting. It avoids the need to link to yet-another-doc but it also means we have to remember to update all of them.

We could use an `include` for this so we don't forget to update it in all the places.

Reply:
Here's an example:
- include
https://github.com/GoogleChrome/web.dev/pull/828/files#diff-813605497e600530c4813a9bc29308a7R1
- usage
https://github.com/GoogleChrome/web.dev/pull/828/files#diff-e9ddf781df35dbafc6749fcf331cc8dcR27

What do you think?

Reply:
Yeah an include is the right thing to do, but I'm gonna add this to the todo list for after the first push.

Reply:
Actually, scratch that. I made some changes to the include and now am including it!